conf/test_full.config

```diff
@@ -11,6 +11,10 @@ params {
   config_profile_name = 'Full test profile'
   config_profile_description = 'Full test dataset to check pipeline function'
 
-  // Input data
+  // Input data for full size test
+  // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
```

Comment:
Can be removed. Guess that for the full test you should also use a bigger dataset and not the same as for the small test.

But I assume that you will do this later anyways - just sayin' now.